src/images/google_cloud_sdk.rs

```diff
@@ -162,6 +125,51 @@ impl CloudSdk {
         }
     }
 
+    /// RunnableImage::from() uses default Image::Args, which is not
+    /// applicable for CloudSdk image. Hence, this helper method can be
+    /// used to set the correct args.
+    pub fn as_runnable(self) -> RunnableImage<Self> {
+        let args = self.arguments.clone();
+        RunnableImage::from(self).with_image_args(args)
+    }
```

Comment:
1. If we can get the `impl Into<RunnableImage<I>> for I` working, I believe we don't need this.
2. If we need this, we should adhere to naming conventions for conversion functions: https://rust-lang.github.io/api-guidelines/naming.html#ad-hoc-conversions-follow-as_-to_-into_-conventions-c-conv

Reply:
Sorry, I got confused. We need this because the `Image` instance no longer carries the `Args` right?

Can we add an `impl From(Image, Image::Args) for RunnableImage` and have the constructors on GoogleCloudSdk returna tuple?